### PR TITLE
Update add-runtime.md

### PR DESCRIPTION
**Pull request by tusharxoxoxo**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

**Comment by tessamero** [comment]
line 93 was fine originally
line 240 was also fine originally

Not able to approve this

File: docs/tutorials/add-runtime.md [hunk 1]
```diff
@@ -90,13 +90,13 @@ The `Response` class must have two functions.
 For interpreted languages use the `path` and `file` parameters to find the file and require it.
 Please make sure to add appropriate checks to make sure the imported file is a function that you can execute.
 
-5. Finally execute the function and handle whatever response the user's code returns. Try to wrap the function into a `try catch` statement to handle any errors the user's function encounters and return them cleanly to the executor with the error schema.
+5. Finally execute the function and handle whatever response the user's code returns. Try to wrap the function into a `try catch ` statement to handle any errors the user's function encounters and return them cleanly to the executor with the error schema.
```
Comment (gewenyu99) [inline docs/tutorials/add-runtime.md:93]: Please revert this change.

File: docs/tutorials/add-runtime.md [hunk 2]
```diff
@@ -237,7 +237,7 @@ Once you have found this, Add your own entry into this array like so:
     'entrypoint'  =>  'Test file', // Replace with the name of the test file you wrote in ./tests/resources/LANGUAGE_NAME
     'timeout'  =>  15,
     'runtime'  =>  'LANGUAGE_NAME-VERSION',
-    'tarname'  =>  'LANGUAGE_NAME-VERSION.tar.gz', // Note: If your version has a point in it replace it with a dash instead for this value.
+    'tarname'  =>  'LANGUAGE_NAME-VERSION.tar.gz', // Note: If your version has a point in it replace it, with a dash instead for this value.
```
Comment (gewenyu99) [inline docs/tutorials/add-runtime.md:240]: Please revert this change :)

**Review by gewenyu99 — CHANGES_REQUESTED** [review]

File: docs/tutorials/add-runtime.md (hunk 1 above)
Comment (tusharxoxoxo) [reply docs/tutorials/add-runtime.md]: done

File: docs/tutorials/add-runtime.md (hunk 2 above)
Comment (tusharxoxoxo) [reply docs/tutorials/add-runtime.md]: reverted

**Comment by Meldiron** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

**Comment by stnguyen90** [comment]
@tusharxoxoxo, looks like there are merge conflicts. Would you please resolve them and then re-request a review.

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**